data/tasks.txt

```diff
@@ -0,0 +1,2 @@
+E@false@book fair@11/01/2000 1000@12/01/2000 1200
+D@false@task a@11/11/2000 1900
```

Comment:
Would you add this file to .gitignore so that the user data would not be revealed?

Reply:
Thank you for catching that! It would indeed be bad if the user’s data is leaked